lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java

```diff
@@ -207,13 +227,18 @@ private void migrateIdentity(
             ReportUserIdentityItem reportItem, PageSummary pageSummary, String batchId) {
         // Skip any non P2 identities
         if (!Vot.P2.name().equals(reportItem.getIdentity())) {
-            // Send skipped audit event
             LOGGER.info(
                     annotateLog(
                             "Skipping migration - not a P2 identity",
                             reportItem,
                             batchId,
                             pageSummary));
+            auditService.sendAuditEvent(
+                    AuditEvent.createWithoutDeviceInformation(
+                            AuditEventTypes.IPV_EVCS_MIGRATION_SKIPPED,
+                            batchId,
+                            new AuditEventUser(reportItem.getHashUserId(), "", "", ""),
```

Comment:
I think in the audit events we should be using the users actual userID rather than the hashed value. That way audit events can be tied together to form more complete journeys.

Also wondering if it might be better to put a placeholder instead of these empty strings - just to make it clear why we don't have values for them.

Reply:
Same for all the audit messages added here.

Reply:
Thanks